src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -130,6 +132,7 @@ protected override Task InitializeAsync(CancellationToken cancellationToken, IPr
         {
             AddService(typeof(IGitHubServiceProvider), CreateService, true);
             AddService(typeof(IUsageTracker), CreateService, true);
+            AddService(typeof(ILoginManager), CreateService, true);
```

Comment:
Could this be exposed as MEF export rather than a Visual Studio service? If we're ever going to add support for Blend (which also hosts TeamExplorer), we'll need to avoid using custom services.